local-dev/importLocalSslCertToJavaKeystore.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/zsh
+set -e
+
+# Define variables
+domain="api-local.fractal.cloud"
+port=8443
+#pathToJavaKeystore="$JAVA_HOME/lib/security/cacerts"
+pathToJavaKeystore="/Users/kamil/Library/Java/JavaVirtualMachines/temurin-21.0.3/Contents/Home/lib/security/cacerts"
```

Comment:
I think we should change this to something that is more generic.
Moreover, we should change the Readme in order to explain how, when, and why to use this script